R/pkg/R/DataFrame.R

```diff
@@ -510,9 +510,7 @@ setMethod("registerTempTable",
 #'
 #' Insert the contents of a SparkDataFrame into a table registered in the current SparkSession.
 #'
-#' @param x A SparkDataFrame
-#' @param tableName A character vector containing the name of the table
```

Comment:
The reasons are

1. It's generic counterpart has `...` and we have to document for that.

2. I think it may be preferred that `...` follows other explicit arguments in order?

Does that make sense?